plugin.yaml

```diff
@@ -597,6 +597,12 @@ dsl_definitions:
       type: boolean
       default: false
 
+  verify_external_resource: &verify_external_resource
```

Comment:
Cannot we depend on `use_external_resource` do we need to add this ?

Reply:
The problem here is I am not sure right now that this method can be used with every resource type.